.github/workflows/test-suite.yml

```diff
@@ -6,7 +6,7 @@ on:
     inputs:
       terraform_matrix:
         description: 'Terraform version matrix (JSON array)'
-        default: '["1.8.x", "1.7.x", "1.6.x", "1.5.x", "1.4.x", "1.3.x", "1.2.x"]'
```

Comment:
Is this change intentional? Removing 1.5.x down to 1.2.x ?

Reply:
my bad, reverted, thx